torch/_torch_docs.py

```diff
@@ -6241,3 +6241,47 @@ def parse_kwargs(desc):
     >>>                            [7, 8, 9]]))
     (tensor([1, 2, 3]), tensor([4, 5, 6]), tensor([7, 8, 9]))
 """)
+
+add_docstr(torch.one_hot,
+           r"""
+one_hot(tensor, num_classes=0) -> LongTensor
+
+Takes integer with :math:`n` dims and converts it to 1-hot representation
```

Comment:
nit: "Takes integer tensor".

Could you add that it takes a tensor of shape `(*)` and returns a tensor of shape `(*, num_classes)` ?